py/torch_tensorrt/dynamo/lowering/passes/__init__.py

```diff
@@ -0,0 +1,11 @@
+from torch.fx.passes.pass_manager import PassManager
+
+from .constant_folding import constant_fold
+from .repair_input_as_output import repair_input_as_output
+
+ATEN_LOWERING_PASSES = PassManager.build_from_passlist(
+    [
+        constant_fold,
+        repair_input_as_output,
+    ]
+)
```

Comment:
`add_lowering_pass` (function), `get_aten_lowering_passes` (function to get list of passes)

Reply:
+ Tutorial on how to build a lowering pass